registrations/models.py

```diff
@@ -146,6 +146,36 @@ def fire_language_metric(sender, instance, created, **kwargs):
         })
 
 
+@receiver(post_save, sender=Registration)
+def fire_source_metric(sender, instance, created, **kwargs):
+    """
+    Fires metrics for each source for each subscription, a sum metric for
+    the registrations over time, and a last metric for the total count.
+    """
+    from .tasks import fire_metric
+    if (created):
+        if "hw" in instance.source.authority:
+            source = 'hcw'
+            sources = ['hw_limited', 'hw_full']
+        else:
+            source = 'public'
+            sources = ['patient', 'advisor']
```

Comment:
The ticket just says `total registrations on Public line`. I decided on this split because in tasks where the welcome message is sent, it sends the health care worker text if the source contains `hw` else it sends the public one. Does the advisor use the hcw or public line?